akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -315,6 +315,10 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
   val Event: FSM.Event.type = FSM.Event
   val StopEvent: FSM.StopEvent.type = FSM.StopEvent
 
+  object → {
```

Comment:
Hah, you're right - it adds the `public abstract akka.actor.FSM<S, D>.$u2192$ $u2192();` method of course, which breaks people extending FSM... :disappointed: